easybuild/easyconfigs/h/HDF5/HDF5-1.8.18-gimkl-2017a.eb

```diff
@@ -0,0 +1,31 @@
+name = 'HDF5'
+version = '1.8.18'
+
+homepage = 'http://www.hdfgroup.org/HDF5/'
+description = """HDF5 is a unique technology suite that makes possible the management of
+ extremely large and complex data collections."""
+
+toolchain = {'name': 'gimkl', 'version': '2017a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = [
+    'https://support.hdfgroup.org/ftp/HDF5/current18/src/',
+    'https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(versions)/src/'
+]
+
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['dd2148b740713ca0295442ec683d7b1c']
+
+buildopts = 'CXXFLAGS="$CXXFLAGS -DMPICH_IGNORE_CXX_SEEK"'
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('Szip', '2.1'),
+]
+
+# Needed for h5py in Python (at least sometimes)
+modextravars = {
+    'HDF5_DIR': '%(installdir)s',
```

Comment:
@valtandor The comment says it's for `h5py`, not `netcdf`?

My main issue is that this should be done consistently in all HDF5 easyconfigs...
Or, (a lot) better, in the existing HDF5 easyblock (one central change).